### PR TITLE
fix: add rohit8020

### LINKED ISSUE #1446: Add Rohit Gangwar
## Profile Screenshot

Screenshot from 2022-10-02 19-32-33

### PR DESCRIPTION
**Pull request by rohit8020**
resolves #1446

## Checklist for profile addition

- [x] Name the PR `fix: add <your-username>`.
- [x] You've added file named `your-username.json` in the [profiles](https://github.com/EddieHubCommunity/awesome-github-profiles/tree/main/profiles) directory.
- [x] You've linked the PR to the correct issue.

---

## Other additions/changes

- [ ] Changes made in this PR are mentioned below
- [ ] Search previous suggestions before making a new one, as yours may be a duplicate.
- [ ] I acknowledge that all my contributions will be made under the project's license.
- [ ] I've read the [contributing guidelines](https://github.com/EddieHubCommunity/awesome-github-profiles/blob/main/CONTRIBUTING.md)

### CODE REVIEW

File: profiles/rohit8020.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    "image": "https://user-images.githubusercontent.com/76878634/193458103-5e05eea9-05b1-4690-bdb4-47f73a48f038.png",
+    "issueId": 1446,
+    "name": "Rohit Gangwar",
+    "username": "rohit8020"
+}
```
Comment (Panquesito7) [inline profiles/rohit8020.json:6]: Missing an endline. Without this endline, there could be issues with some IDEs and configurations.
```suggestion
}

```

**Review by Panquesito7 — CHANGES_REQUESTED** [review]

**Comment by rohit8020** [comment]
Added an endline in rohit8020.json

**Comment by rohit8020** [comment]
Added an endline in rohit8020.json

**Review by Panquesito7 — APPROVED** [review]
Looks good. Thanks! 🎉

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**